Win32/COrderlyAsyncProducer.vcxproj

```diff
@@ -0,0 +1,117 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
if Win32/COrderlyAsyncProducer.vcxproj and Win32/rocketmq-client-cpp.sln need merge to master ?

Reply:
I agree with you on this issue. Such configuration files shouldn't be merge into master.

Reply:
yes,remove middle-time file will make pure commit,we can do this